unity-renderer/Assets/Scripts/MainScripts/DCL/DataStore/DataStore.cs

```diff
@@ -31,6 +31,7 @@ public class DataStore
         public readonly DataStore_VirtualAudioMixer virtualAudioMixer = new DataStore_VirtualAudioMixer();
         public readonly DataStore_Screen screen = new DataStore_Screen();
         public readonly DataStore_WSCommunication wsCommunication = new DataStore_WSCommunication();
+        public readonly BaseCollection<PluginFeature> loadedPluginFeatures = new BaseCollection<PluginFeature>();
```

Comment:
```suggestion
        public readonly BaseCollection<PluginFeature> loadedPluginFeatures = new BaseCollection<PluginFeature>();
```
Exposing all the `PluginFeature` instances like this can become a can of worms. Plugins shouldn't know about the existence of other plugins, and the inter-plugin communication should be by using the model exposed by each plugin `DataStore`. 

Here we are breaking that pattern because any plugin could just access directly to other one and modify it.

Reply:
Fixed in https://github.com/decentraland/unity-renderer/pull/1193/commits/f1dda506abfb0210bc4b060687d40e40dccef846. Thanks Brian!!